components/grid.js

```diff
@@ -0,0 +1,126 @@
+const gridTemplate = document.createElement('template');
+
+gridTemplate.innerHTML = `
+
+<p class="component-header">Simplified Grid</p>
+<p class="header-description">The grid creates visual consistency between layouts while allowing
+    flexibility across a wide variety of designs.</p>
+
+<p class="sub-header">Example of 50:50 grid layout</p>
+
+<p>Add <span class="nav-U-I blue-text">grid-50-50-layout </span> class on wrapper div which is wrapping 2 divs.</p>
+
+<div class="avatar-container">
+
+    <div class="grid-50-50-layout">
+        <div>
+            <img src="images/pedri-grid.png" alt="" class="image-responsive">
+        </div>
+        <div class="grid-text-div">
+            <h1 class="grid-heading">Pedri, a year of consolidation</h1>
+            <p>Winner of the Kopa and Golden Boy trophies, the Barça midfielder had an outstanding 2021.
+                In addition to the individual awards...</p>
+            <button class="btn btn-contained-primary">Read more</button>
+        </div>
+    </div>
+
+</div>
+
+<p class="sub-header">Example of 70:30 grid layout</p>
+
+<p>Add <span class="nav-U-I red-text">grid-70-30-layout </span> class on wrapper div which is wrapping 2 divs.</p>
+
+<div class="avatar-container">
+
+    <div class="grid-70-30-layout">
+        <div>
+            <img src="images/Alexia-Putellas-grid.png" alt="" class="image-responsive">
+        </div>
+        <div class="grid-text-div">
+            <h1 class="grid-heading">Alexia Putellas, an unforgettable 2021</h1>
+            <p>The Barça captain has been one of the world's standout footballers in 2021. The Catalan
+                midfielder...</p>
+            <button class="btn btn-text-primary card-btn">READ MORE</button>
+        </div>
+    </div>
+
+</div>
+
+<p class="sub-header">Example of 4 column grid layout</p>
+
+<p>Add <span class="nav-U-I red-text">grid-4-column-layout </span> class on wrapper div which is wrapping 2 divs.</p>
```

Comment:
Can change this class name `nav-U-I`

Reply:
yes, actually I've used it at too many places to change now, do you have any idea where I can change them all at once?

Reply:
`Search & replace all` feature you can try in VS code